src/components/institution/UnconfirmedOrganizationBox.tsx

```diff
@@ -0,0 +1,58 @@
+import ErrorIcon from '@mui/icons-material/Error';
+import SearchIcon from '@mui/icons-material/Search';
+import { Box, BoxProps, Button, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { DeleteIconButton } from '../../pages/messages/components/DeleteIconButton';
+import { dataTestId } from '../../utils/dataTestIds';
+import { StyledOrganizationBox } from './OrganizationBox';
+
+interface UnconfirmedOrganizationBoxProps extends Pick<BoxProps, 'sx'> {
+  name: string;
+  onIdentifyAffiliationClick: (name: string) => void;
+  removeAffiliation?: () => void;
+}
+
+export const UnconfirmedOrganizationBox = ({
+  name,
+  sx,
+  onIdentifyAffiliationClick,
+  removeAffiliation,
+}: UnconfirmedOrganizationBoxProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <StyledOrganizationBox
+      sx={{
+        ...sx,
+      }}>
```

Comment:
kan se ut som dette er overflødig?
```suggestion
    <StyledOrganizationBox>
```